plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue

```diff
@@ -485,7 +529,10 @@ export default defineComponent({
         this.resetPasswordVar();
         this.showUserCreatedNotification();
         this.$emit('updated', { user: this.theUser });
-      });
+      } catch (e) {
+        this.isSavingUserInfo = false;
+        throw e;
+      }
```

Comment:
If the exception is thrown anyway, I would suggest using `finally` instead
```suggestion
      } finally (e) {
        this.isSavingUserInfo = false;
      }
```